src/components/users/UserSupervisorEditable.tsx

```diff
@@ -0,0 +1,73 @@
+import { ComboBox, Input, MenuItem } from '@taskany/bricks';
+import { gray10, gray7 } from '@taskany/colors';
+import styled from 'styled-components';
+import { useState } from 'react';
+import { User } from 'prisma/prisma-client';
```

Comment:
We have issue about ban to do import of prisma interfaces in client code.

Reply:
Enabling the rule will trigger large scale refactoring, which will be outside the scope of this issue. It should be done separately
https://github.com/taskany-inc/crew/issues/288